test/rules/prefer_initializing_formals.dart

```diff
@@ -228,3 +228,39 @@ class GoodCaseWithDifferentNamedArgsInitializer {
       : this.x = a, // OK
         this.y = b; // OK
 }
+
+class GoodCaseWithDifferentType {
+  Iterable<int> ids;
+
+  GoodCaseWithDifferentType(List<int> ids) {
+    this.ids = ids; // OK
+    ids.add(12);
+  }
+}
+
+class GoodCaseWithDifferentTypeNamed {
+  Iterable<int> ids;
+
+  GoodCaseWithDifferentTypeNamed({List<int> ids}) {
```

Comment:
Was this intended to be a named constructor? (I'm not sure what "Named" means in this context.)

Reply:
Sorry my bad it was actually NamedParams updated